scripts/src/main/resources/scripts/command/az

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+function doSetup() {
+  if doIsAzureInstalled
+  then
+    if [ "${1}" != "silent" ] && ! doIsQuiet
+    then
+      doEcho "Azure CLI is already installed at $(command -v az)"
+    fi
+  else
+    doRequireNotBatch
+    if doIsWindows
+    then
+      # Get leatest release
+      if [ -z "${AZURECLI_VERSION}" ]
+      then
+        doEcho "Getting latest release..."
+        AZURECLI_VERSION=$(curl -s https://api.github.com/repos/Azure/azure-cli/releases/latest | awk -F ":" '/tag_name/ {print $2}'| awk -F "\"" '{print $2}' | awk -F "-" '{print $3}')
+      fi
+      local version="${AZURECLI_VERSION}"
+      doDownload "-" "" "az" "${version}"
+      # Install Azure CLI for Windows
+      doEcho "Installing Azure CLI for Windows..."
+      modifyPathStructure
+      installOnWindows
+    else doFail "Sorry, Azure CLI installation support is not yet implemented for your OS. Please install manually or help devonfw-ide to support it for your OS by contributing a pull-request."
+    fi
+  fi
+}
+
+# modify a filepath for powershell path syntax e.g. /c/... -> c:/...
+function modifyPathStructure(){
+  modifiedPath=${DEVON_DOWNLOAD_DIR}
+  partition=$(echo "${modifiedPath}" | cut -f2 -d'/'):'/' # -> <partition>:/
+  modifiedPath=${partition}$(echo "${modifiedPath}" | cut -f3- -d'/')
+  modifiedPath="${modifiedPath////\\}"
+}
```

Comment:
here we should discuss as a team. Others had the same issue for git-bash and also we had solved this for cygwin years ago. IMHO we should have a function `doConvertPathToOs` or something that could solve this issue once and for all (different OS, shells).

Reply:
as @CREITZ25 has figured out we can always use `cygpath` in bash on windows as both cygwin and git-bash do have this tool preinstalled. This will work more reliably than any custom replacement hacks.

Reply:
> as @CREITZ25 has figured out we can always use `cygpath` in bash on windows as both cygwin and git-bash do have this tool preinstalled. This will work more reliably than any custom replacement hacks.

That's really cool! Thank you.